jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java

```diff
@@ -100,7 +100,7 @@ private VolumeBindingUtil() { }
      * Resolves relative paths in the supplied {@code bindingString}, and returns a binding string that has relative
      * paths replaced with absolute paths.  If the supplied {@code bindingString} does not contain a relative path, it
      * is returned unmodified.
-     * <h3>Discussion:</h3>
+     * <h4>Discussion:</h4>
```

Comment:
/home/circleci/project/jkube-kit/build/service/docker/target/generated-sources/delombok/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103: error: header used out of sequence: <H4>
[ERROR]      * <h4>Discussion:</h4>

Reply:
```
[ERROR] /home/circleci/project/jkube-kit/build/service/docker/target/generated-sources/delombok/org/eclipse/jkube/kit/build/service/docker/helper/VolumeBindingUtil.java:103: error: header used out of sequence: <H4>
[ERROR]      * <h4>Discussion:</h4>
[ERROR]        ^
```

OMG different Java versions are complaining of different settings. Locally with Java17 had the same error with H3. 
Not a good idea to add titles to Javadocs removing them...

Reply:
Sounds good to remove it.